VogtPayroll3/PayrollManager.cs

```diff
@@ -0,0 +1,130 @@
+                Console.WriteLine("Gross Pay: " + employee.GetGrossPay());
+                Console.WriteLine("Deductions: " + employee.GetDeductions());
+                Console.WriteLine("Net Pay: " + employee.GetNetPay());
+            }
+        }
+
+        public Dictionary<int, Employee> RemoveUserInDictionary(Dictionary<int, Employee> dictionary, int empID)
+        {
+            //var dictionary = empList.ToDictionary(x => x.EmpID);
+
+            Employee employeeTemp = dictionary[empID];
+
+            dictionary.Remove(employeeTemp.EmpID);
+
+            if (dictionary.TryGetValue(employeeTemp.EmpID, out employeeTemp))
+            {
+                Console.WriteLine($"For key = {employeeTemp.EmpID}, value = {employeeTemp.FirstName}.");
+            }
+            else
+            {
+                Console.WriteLine($"Key = {empID} is not found.");
+            }
+
+            return dictionary;
+
+        }
+        public Dictionary<int, Employee> UpdateUserInDictionary(List<Employee> empList, int empID)
+        {
+            var dictionary = empList.ToDictionary(x => x.EmpID);
+
+            PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
+            Employee employeeTemp = dictionary[empID];
+
+            employeeTemp.HoursWorked = payrollConsoleReader.GetHoursWorkedConsole();
+            employeeTemp.Payrate = payrollConsoleReader.GetPayrateConsole();
+
+            dictionary.Add(employeeTemp.EmpID, employeeTemp);
+
+            return dictionary;
+        }
+
+        public Dictionary<int, Employee> ChangeUserInDictionary(List<Employee> empList, int empID)
+        {
+
+            var dictionary = empList.ToDictionary(x => x.EmpID);
+
+            PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
+            Employee employeeTemp = dictionary[empID];
+
+            employeeTemp.FirstName = payrollConsoleReader.GetFirstNameConsole();
+            employeeTemp.LastName = payrollConsoleReader.GetLastNameConsole();
+            //employeeTemp.EmpID = payrollConsoleReader.GetEmployeeIDConsole();
+            employeeTemp.HoursWorked = payrollConsoleReader.GetHoursWorkedConsole();
+            employeeTemp.Payrate = payrollConsoleReader.GetPayrateConsole();
+
+            dictionary.Add(employeeTemp.EmpID, employeeTemp);
+
+            return dictionary;
+        }
+
+        public Dictionary<int, Employee> DisplayEmployeeInformationDictionary(Dictionary<int, Employee> dictionary)
+        {
+
+            
+            Console.WriteLine("\nKeys and Values: ");
+
+            foreach (KeyValuePair<int, Employee> employeeKeyValuePair in dictionary)
+            {
+                try
+                {
+                    Console.WriteLine("Key: " + employeeKeyValuePair.Key);
+                    Employee emp = employeeKeyValuePair.Value;
+                    Console.WriteLine("Value: " + emp.FirstName + " " + emp.LastName);
```

Comment:
Why are you wrapping this in a try catch? Usually, iterating through a dictionary is a relatively safe operation.